README.md

```diff
@@ -20,16 +20,16 @@ Configuration is achieved by specifying paths to various certificates private ke
 
 ```bash
 # Pass certificate
-export SIGNING_CERT=/path/to/cert.pem
+export SIGNING_CERT=<...>
```

Comment:
I'd make a note above about which fields are Base64.
Does the password have to be Base64?